app/components/gh-nav-menu.js

```diff
@@ -11,13 +11,15 @@ export default Component.extend({
 
     open: false,
 
-    navMenuIcon: computed('config.blogUrl', function () {
-        let url = `${this.get('config.blogUrl')}/favicon.png`;
+    navMenuIcon: computed('config.blogUrl', 'settings.icon', function () {
+        let blogIcon = this.get('settings.icon') ? this.get('settings.icon') : 'favicon.ico';
```

Comment:
is `settings.icon` always null/undefined when there's no icon or does is it get set to a blank string? It might be worth wrapping it in an `isBlank()` anyway